### PR TITLE
Basic trainer notebook with the specifications

### PR DESCRIPTION
**Pull request by sayakpaul**
Specifications followed from here: 

https://github.com/keras-team/keras-io/blob/master/call_for_contributions.md#large-scale-multi-label-text-classification

### CODE REVIEW

File: multi_label_trainer.ipynb [hunk 1]
```diff
@@ -0,0 +1,818 @@
+{
```
Comment (soumik12345) [inline multi_label_trainer.ipynb:1]: <p>Before loading the data, once we should check if there are any redundant titles in the dataset or not. The search queries we used to scrape the data could infact overlap.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

Comment (sayakpaul) [reply multi_label_trainer.ipynb:1]: <p>Or should we do that after loading the dataframe? </p><p> </p><p>If before, then I suggest it's done before we serialize the initial dataframe. </p>

Comment (soumik12345) [reply multi_label_trainer.ipynb:1]: I thinkk we shoul do it before serialization.

Comment (soumik12345) [reply multi_label_trainer.ipynb:1]: I'll update the data scraping notebook with the necessary pre-processing.

Comment (sayakpaul) [reply multi_label_trainer.ipynb:1]: <p>Alright. </p>

**Comment by sayakpaul** [comment]
@soumik12345 incorporated the new dataset and added minor changes for enhancement.

**Comment by soumik12345** [comment]
@sayakpaul changes seem good to go.

**Pull request merged**